core/templates/dev/head/components/answer_group_editor.html

```diff
@@ -112,6 +112,7 @@
     <form role="form" class="form-inline protractor-test-edit-outcome-dest" name="editAnswerGroupForm.editDestForm" ng-submit="saveThisDestination()">
       <outcome-destination-editor outcome="outcome">
       </outcome-destination-editor>
+      <p ng-show="editAnswerGroupForm.editDestForm.$invalid && editAnswerGroupForm.editDestForm.newStateName.$dirty" class="help-block oppia-form-error" style="font-size: 0.85em;">Please pick a card name consisting of alphanumeric characters, spaces and/or hyphens only.</p>
```

Comment:
I tried to put it inside and the message could not be displayed at all. so i had to used [nested form](https://docs.angularjs.org/api/ng/directive/ngForm) and it worked.
Done!